src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -996,6 +996,57 @@ def migration_check_name_availability(cmd, client, resource_group_name, server_n
     return get_postgresql_flexible_management_client(cmd.cli_ctx).check_migration_name_availability(subscription_id, resource_group_name, server_name, migration_name_availability_parammeters)
 
 
+def virtual_endpoint_create_func(client, resource_group_name, server_name, virtual_endpoint_name, endpoint_type, members):
+    parameters = {
+        'name': virtual_endpoint_name,
+        'endpoint_type': endpoint_type,
+        'members': [members]
```

Comment:
Please check if it throws error if you do not pass some of the parameters for the command scope. If error comes directly from SDK / backend it is better to add client side validation

Reply:
I am getting errors thrown before it reaches these functions:
<img width="1322" alt="image" src="https://github.com/Azure/azure-cli/assets/69922333/000b7a67-1823-45db-8d94-1464ef4ec451">

Reply:
Not required, idea is that it should not go to controller for missing parameters